utils/parametric/_library_client.py

```diff
@@ -201,6 +203,8 @@ def otel_trace_start_span(
                 "attributes": attributes or {},
             },
         ).json()
+        span_id = resp["span_id"]
+        print(f"span_id is in start_span is {span_id}")
```

Comment:
better to use logger: `from utils.tools import logger`

Reply:
ah, sorry about leaving those in, thanks for the tip!